etc/mqtt_source.yaml

```diff
@@ -15,3 +15,6 @@ default:
 demo_conf: #Conf_key
   qos: 0
   server: "tcp://10.211.55.6:1883"
+
+baetyl_conf: # baetyl_broker_conf_key
```

Comment:
Looks like you only accept the user to use a single connection instance? If so, this is not needed. Otherwise, it may be better to copy all props here instead of refer to the connection selector